mlflow/utils/run_operations.py

```diff
@@ -0,0 +1,27 @@
+class RunOperations:
+    """
+    Represents a collection of operations on one or more MLflow Runs, such as run creation
+    or metric logging.
+    """
+
+    def __init__(self, operation_futures):
+        self._operation_futures = operation_futures or []
+
+    def await_completion(self):
```

Comment:
Can we call this `wait()`? This is consistent with Python futures and MLflow Projects (https://docs.python.org/3/library/concurrent.futures.html#concurrent.futures.wait)